sbt/src/sbt-test/compiler-project/dotty-compiler-plugin/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.1.7")
```

Comment:
Will be `0.2.0` once released for sbt1

Reply:
Thanks @allanrenucci , looking forward to 0.2.0 release .